src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -977,6 +975,29 @@ using namespace ngraph::runtime;
     }
 }
 
+bool runtime::cpu::CPU_ExternalFunction::is_output(const ngraph::descriptor::Tensor& tensor)
+{
+    return m_variable_name_map[tensor.get_name()].find("output") != std::string::npos;
+}
```

Comment:
Would be good to create a const static string for "outputs" and use it here and at line 672, so folks dont accidentally break this. Also rename function to 'is_output_buffer'?